.gitignore

```diff
@@ -16,8 +16,12 @@ out/
 
 # Eclipse
 /.project
+**/.project
```

Comment:
[Q] Did you intend to add this gitignore change to the PR?

IMO this should be its own ticket since it's unrelated to the main thrust of this PR.

In the meantime, if this stuff is causing you pain as you develop, you could add it to your machine's ["global" gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
don't worry about this one, I have opened a different PR for Eclipse things https://github.com/apache/solr/pull/1357 just forgot to remove it from here